Commands/Rubocop.tmCommand

```diff
@@ -12,6 +12,8 @@ if [ -z "$TM_RUBY" ]; then
     exit 1
 fi  
 
+cd "$TM_PROJECT_DIRECTORY"
+[ -z "$TM_RUBOCOP_RUBY_VERSION" ] && export TM_RUBOCOP_RUBY_VERSION=2.2
```

Comment:
My Textmate could not load the bundle with this line present. I set the Textmate variable, removed the line and everything works as expected.